doc/man7/migration_guide.pod

```diff
@@ -0,0 +1,1814 @@
+
+=item EC_KEY_get0_group, EC_KEY_get0_private_key, EC_KEY_get0_public_key,
+EC_KEY_get_conv_form and EC_KEY_get_enc_flags
+
+See L</Deprecated low-level key parameter getters>.
+
+=item EC_KEY_get0_engine, EC_KEY_get_default_method, EC_KEY_get_method,
+EC_KEY_new_method, EC_KEY_get_ex_data, EC_KEY_OpenSSL, EC_KEY_set_ex_data,
+EC_KEY_set_default_method, EC_KEY_METHOD_*, EC_KEY_set_method
+
+See L</Providers are a replacement for engines and low-level method overrides>
+
+=item EC_METHOD_get_field_type
+
+Use L<EC_GROUP_get_field_type(3)> instead.
+See L</Providers are a replacement for engines and low-level method overrides>
+
+=item EC_KEY_key2buf, EC_KEY_oct2key, EC_KEY_oct2priv, EC_KEY_priv2buf and EC_KEY_priv2oct
+
+There are no replacements for these.
+
+=item EC_KEY_new, EC_KEY_new_by_curve_name, EC_KEY_free, EC_KEY_up_ref
+
+See L</Deprecated low-level object creation>
+
+=item EC_KEY_print, EC_KEY_print_fp
+
+See L</Deprecated low-level key printing functions>
+
+=item EC_KEY_set_asn1_flag, EC_KEY_set_conv_form, EC_KEY_set_enc_flags
+
+See L</Deprecated low-level key parameter setters>.
+
+=item EC_KEY_set_group, EC_KEY_set_private_key, EC_KEY_set_public_key, EC_KEY_set_public_key_affine_coordinates
+
+See L</Deprecated low-level key parameter setters>.
+
+=item ECParameters_print, ECParameters_print_fp, ECPKParameters_print and
+ECPKParameters_print_fp
+
+See L</Deprecated low-level key printing functions>
+
+=item EC_POINT_bn2point and EC_POINT_point2bn
+
+These functions were not particularly useful, since EC point serialization
+formats are not individual big-endian integers.
+
+=item EC_POINT_get_affine_coordinates_GF2m, EC_POINT_get_affine_coordinates_GFp,
+EC_POINT_set_affine_coordinates_GF2m and EC_POINT_set_affine_coordinates_GFp
+
+Applications should use L<EC_POINT_get_affine_coordinates(3)> and
+L<EC_POINT_set_affine_coordinates(3)> instead.
+
+=item EC_POINT_get_Jprojective_coordinates_GFp and EC_POINT_set_Jprojective_coordinates_GFp
+
+These functions are not widely used. Applications should instead use the
+L<EC_POINT_set_affine_coordinates(3)> and L<EC_POINT_get_affine_coordinates(3)>
+functions.
+
+=item EC_POINT_make_affine and EC_POINTs_make_affine
+
+There is no replacement. These functions were not widely used, and OpenSSL
+automatically performs this conversion when needed.
+
+=item EC_POINT_set_compressed_coordinates_GF2m and EC_POINT_set_compressed_coordinates_GFp
+
+Applications should use L<EC_POINT_set_compressed_coordinates(3)> instead.
+
+=item EC_POINTs_mul
+
+This function is not widely used. Applications should instead use the
+L<EC_POINT_mul(3)> function.
+
+=item ENGINE_*
+
+All engine functions are deprecated. An engine should be rewritten as a provider.
+See L</Providers are a replacement for engines and low-level method overrides>.
+
+=item ERR_func_error_string
+
+=item ERR_get_error_line
+
+=item ERR_get_error_line_data
+
+=item ERR_get_state
+
+=item ERR_load_*
```

Comment:
These would be better as a comma separated list like all of the earlier items.

Reply:
It is debatable - There is an ENGINE_*.. Not sureit gains us much (This was alphabetically ordered - at least as much as I could).

Reply:
These should collapse at the least:
```POD
=item ERR_func_error_string
=item ERR_get_error_line
=item ERR_get_error_line_data
=item ERR_get_state
```